hackage-server.cabal

```diff
@@ -94,6 +96,7 @@ flag cabal-parsers
   manual: True
   default: False
 
+build-tool-depends: cabal-doctests
```

Comment:
I don't think `build-tool-depends` can be at the top-level, it is like `build-depends`.